src/Symfony/Component/Notifier/Bridge/Octopush/README.md

```diff
@@ -0,0 +1,25 @@
+Octopush Notifier
+===============
+
+Provides [Octopush](https://www.octopush.com) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+OCTOPUSH_DSN=octopush://USERLOGIN:APIKEY@default?from=FROM&sms_type=SMSTYPE
+```
+
+where:
+- `USERLOGIN` is your Octopush email
```

Comment:
If user is an email (and contains an `@`) I believe it should be url_encoded. 
IMHO we should warn the user about this

Reply:
Shall we rename it to `EMAIL` then?

Reply:
It is possible to rename the field by EMAIL if you prefer. I hesitated to do it and I preferred to follow the name provided by the Octopush documentation called 'user_login' to keep consistency with other parameters such as the 'sms_type' which is also required : https://www.octopush.com/api-sms-documentation/parametres/